guides-markdown/vps-setup.md

```diff
@@ -0,0 +1,338 @@
+# Full Node VPS Setup
+
+```post-author
+Alex Sherbuck
+```
+
+```post-description
+This guide provides installation instructions for a Bcoin full node running on a Digital Ocean of Amazon AWS Virtual Private Server.
+```
+
+
+* [Introduction](#introduction)
+* [Digital Ocean](#digital-ocean)
+* [Amazon AWS](#amazon-aws)
+* [Connecting to your server](#connecting-to-your-server)
+* [Setting up the environment](#setting-up-the-environment)
+* [Create a New User](#create-a-new-user)
+* [Install Bcoin](#install-bcoin)
+
+
+## Introduction
+
+Running a full node requires your computer always be online and connected to the Bitcoin network. For most users, a VPS is an elegant 24/7 full node solution.
+
+## Digital Ocean
+
+### Create an account @digitalocean.com
+### Choose 'Create' -> 'Droplet' from the account dashboard
+
+![create droplet](https://raw.githubusercontent.com/tenthirtyone/bcoin-org.github.io/staging/assets/images/guides/vps-create-droplet.png "Create Droplet")
+### Choose 'Ubuntu 16.04' for OS Distribution
+![choose os](https://raw.githubusercontent.com/tenthirtyone/bcoin-org.github.io/staging/assets/images/guides/vps-choose-os.png "Choose OS")
```

Comment:
To avoid linking to your github repo, you should just be able to do this as a relative path: `../assets/images/`